examples/config-api/config_api.bal

```diff
@@ -20,37 +20,36 @@ import ballerina/log;
 //
 // The configuration file format is a subset of the TOML file format. It
```

Comment:
Shall we have all value types, etc. between single back ticks in this example?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/12036/commits/837cb3cd6d7fbda163111adff81befa01a0fb07d